Cargo.toml

```diff
@@ -21,9 +21,10 @@ keccak-hash = "0.10.0"
 
 # zk-evm dependencies
 plonky2 = "0.2.2"
-evm_arithmetization = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-trace_decoder = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
-proof_gen = { git = "https://github.com/0xPolygonZero/zk_evm.git", tag = "v0.3.1" }
+evm_arithmetization = { git = "https://github.com/fractal-zkp/zk_evm.git", branch = "trace-processing-new" }
```

Comment:
We've merged your `zk_evm` PR now

Reply:
Ok I'll integrate those changes.

Reply:
I've updated to point to the develop branch of `zk_evm`. If we want to use a tag then we need to cut a new tag on `zk_evm`.

Reply:
Yep we intend to do it next week.